AssetRipper.GUI.SourceGenerator/Program.cs

```diff
@@ -4,13 +4,16 @@ namespace AssetRipper.GUI.SourceGenerator;
 
 public static class Program
 {
-	public static void Main(string[] args)
+	private const string repositoryPath = "../../../../";
+
+	private static string GetPathForProject(string project)
 	{
-		string repositoryPath = "../../../../";
-		string outputPath = Path.Combine(repositoryPath, "AssetRipper.GUI", "LocalizationManager.g.cs");
-		
-		string source = LocalizationSourceGenerator.MakeLocalizationClass(repositoryPath);
+		return Path.Combine(repositoryPath, project);
+	}
 
-		File.WriteAllText(outputPath, source);
+	public static void Main(string[] args)
+	{
+		string localizationPath = Path.Combine(GetPathForProject("AssetRipper.GUI"), "LocalizationManager.g.cs");
+		File.WriteAllText(localizationPath, LocalizationSourceGenerator.MakeLocalizationClass(repositoryPath));
```

Comment:
This seems completely unrelated to everything else you were doing.

Reply:
Oh yeah this was some refactoring I started doing in anticipation of writing some more codegen for the feature that validates that valid enum values are passed. I this this is more future proof anyway, but since I ended up not writing new codegen functions, it's an orphaned change.